gym/envs/__init__.py

```diff
@@ -377,6 +377,13 @@ def _merge(a, b):
         max_episode_steps=100,
     )
 
+    register(
```

Comment:
Could you please also register the environments for `Egg` and `Pen` with touch sensing here?